config/jobs/kubernetes-sigs/kueue/kueue-release-blocking.yaml

```diff
@@ -0,0 +1,245 @@
+          securityContext:
+            privileged: true
+  - interval: 1d
+    name: periodic-kueue-test-e2e-main-1-25
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-kueue-test-e2e-main-1-25
+      description: "Run periodic kueue end to end tests for Kubernetes 1.25"
+      testgrid-num-columns-recent: '6'
+      testgrid-num-failures-to-alert: "1"
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: kueue
+        base_ref: master
+        path_alias: kubernetes-sigs/kueue
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230222-b5208facd4-master
+          env:
+            - name: E2E_KIND_VERSION
+              value: kindest/node:v1.25.3
+          command:
+            # generic runner script, handles DIND, bazelrc for caching, etc.
+            - runner.sh
+          args:
+            - make
+            - kind-image-build
+            - test-e2e
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
+  - interval: 1d
+    name: periodic-kueue-verify-main
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-kueue-verify-main
+      description: "Run periodic kueue verify checks"
+      testgrid-num-columns-recent: '6'
+      testgrid-num-failures-to-alert: "1"
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: kueue
+        base_ref: master
+        path_alias: kubernetes-sigs/kueue
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    spec:
+      containers:
+        - image: golang:1.19
+          command:
+            - make
+          args:
+            - verify
+  - interval: 1d
+    name: periodic-kueue-build-image-main
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-kueue-build-image-main
+      description: "Periodically build container image of kueue"
+      testgrid-num-columns-recent: '6'
+      testgrid-num-failures-to-alert: "1"
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: kueue
+        base_ref: master
+        path_alias: kubernetes-sigs/kueue
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230222-b5208facd4-master
+          securityContext:
+            privileged: true
+          command:
+            - runner.sh
+          args:
+            - make
+            - image-local-build
+  - interval: 1d
+    name: periodic-kueue-build-image-release-0-1-and-release-0-2
```

Comment:
Remove this job. We will not need it.

Reply:
How about pull-kueue-build-image-main.I see we skip release 0.1/0.2 in that test,should we remove that test too?

Reply:
I'm inclined to remove it. As I said, the E2E tests already do a build, so we are not losing coverage.